packages/wmr/src/plugins/npm-plugin/resolve.js

```diff
@@ -23,7 +34,7 @@ export async function resolveModule(path, { readFile, hasFile, module, internal
 	if (!internal && pkg.exports) {
 		const entry = path ? `./${path}` : '.';
 
-		const mapped = resolveExportMap(pkg.exports, entry, ENV_KEYS);
+		const mapped = resolveExports(pkg.exports, entry);
```

Comment:
This should be the full package.json contents
```suggestion
		const mapped = resolveExports(pkg, entry);
```

Reply:
The module will also normalize the `entry` value too: https://github.com/lukeed/resolve.exports/blob/master/src/index.js#L64